packages/dashboard/.env.example

```diff
@@ -1,19 +1,20 @@
 # Client Side Variables -- Anything using NEXT_PUBLIC_ will be readable in the client
-NEXT_PUBLIC_SIDETREE_METHOD="elem:ganache"
+NEXT_PUBLIC_SIDETREE_METHOD="example:sidetree.testnet"
+NEXT_PUBLIC_ELEMENT_METHOD='elem:ganache'
 NEXT_PUBLIC_OPERATOR="Example Corp."
-NEXT_PUBLIC_METHOD="Element Ganache"
-NEXT_PUBLIC_DESCRIPTION="Sidetree on Ethereum Ledger and IPFS Cas."
+NEXT_PUBLIC_METHOD="Sidetree Test Method"
+NEXT_PUBLIC_DESCRIPTION="Sidetree on Mock Ledger and Mock Cas."
 
 # Sidetree Variables
 MONGO_DB_CONNECTION_STRING="mongodb://localhost:27017/"
-DATABASE_NAME="sidetree-ganache-local"
+DATABASE_NAME="testMongodbName"
 MAX_CONCURRENT_DOWNLOADS=20
 BATCH_INTERVAL_IN_SECONDS=1
 OBSERVING_INTERVAL_IN_SECONDS=1
 PORT=3000
 
-# Element Node Variables
-ELEMENT_CONTENT_ADDRESSABLE_STORE_SERVICE_URI='/ip4/127.0.0.1/tcp/5001' # Required if using 'elem:ganache'
-ETHEREUM_RCP_URL='http://localhost:8545' # Required if using 'elem:ganache'
-ETHEREUM_PROVIDER=$ETHEREUM_RCP_URL # Required if using 'elem:ganache'
-# MNEMONIC= # Optional if using 'elem:ganache'
\ No newline at end of file
+# Required if using 'elem:ganache'
+ELEMENT_CONTENT_ADDRESSABLE_STORE_SERVICE_URI='/ip4/127.0.0.1/tcp/5001' 
+ETHEREUM_RCP_URL='http://localhost:8545'
+ETHEREUM_PROVIDER=$ETHEREUM_RCP_URL
+MNEMONIC="hazard pride garment scout search divide solution argue wait avoid title cave"
```

Comment:
I don't think we should show a mnemonic value since it might be used somewhere and it is optional for ganache. E.g.
`# MNEMONIC="your value here" # this is optional for ganache and required for testnet`

Reply:
Copy that! I will updated the PR with your suggestion!